inc/az_ulib_ipc_api.h

```diff
@@ -458,8 +384,8 @@ AZ_NODISCARD az_result az_ulib_ipc_call_with_str(
  *      match the published name and version.
  * 3. The names are case sensitive.
  * 4. If device_name or capability_name is not provided, this function shall return #AZ_SPAN_EMPTY.
- * 5. The package_version can be a number or a wildcard character `*`, this function shall return
- *      `package_version` = #AZ_ULIB_VERSION_DEFAULT for a wildcard character.
+ * 5. The package_version can be a number or a wild-card character `*`, this function shall return
```

Comment:
nit: "... or a wildcard character `*`. This function shall return... "
Also you can use "wildcard". It doesn't need to be hyphenated.

Reply:
#Pending